collector/topological_inventory.py

```diff
@@ -173,30 +175,111 @@ def _query_sub_collection(
     return all_data
 
 
-def worker(_: str, source_id: str, dest: str, b64_identity: str) -> None:
-    """Worker for topological inventory.
+def worker(
+        _: str,
+        tenants_url: str,
+        all_tenants: bool,
```

Comment:
Please don't change the API of the worker, it'll make difficult to propagate changes which would need to hit all AI branches. The topological worker is already loading env data from `collector/env`, why this has to be any different? `collector/env` is much better place for storing `ALL_TENANTS` switch and `TENANTS_URL`, then the server. It's not necessary for all AI services workflow, so it shouldn't modify the shared API.

Reply:
Yes, indeed.
Still getting used to the new scheme of things with the recent code shuffle.

I will address this. Thanks.